src/core/loss_detection.c

```diff
@@ -1305,8 +1305,13 @@ QuicLossDetectionProcessAckBlocks(
                 // All previously considered lost packets were found to be
                 // spuriously lost. Inform congestion control.
                 //
-                QuicCongestionControlOnSpuriousCongestionEvent(
-                    &Connection->CongestionControl);
+                if (QuicCongestionControlOnSpuriousCongestionEvent(
```

Comment:
Will this result in multiple flushes happening back to back? one that actually sends data, and one that is a no-op?

Reply:
Nope